### PR TITLE
Merge from beta to production

### PR DESCRIPTION
**Pull request by PongDev**
## Change made

- [x] Bug fixes
- [x] New features
- [x] Breaking changes

## Describe what you have done

- All requirements for sprint 3

### CODE REVIEW

**Review by TendonT52 — APPROVED** [review]
LGTM

File: apps/backend/src/bookings/bookings.service.ts [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+    account: AccountMetadata,
+  ): Promise<CancelBookingByTravellerResponse> {
+    try {
+      const booking = await this.bookingsRepo.getBookingById(bookingId);
+      console.log({ booking, account });
+      if (booking.userId !== account.userId) {
+        throw new AccessNotGranted('permissing denied');
+      }
+      if (
+        booking.bookingStatus !== 'TRAVELING' &&
```
Comment (MinitrueSecretary) [inline apps/backend/src/bookings/bookings.service.ts:287]: Does this line means the traveller can cancel the trip despite it already been started
sounds weird, but i dunno

**Review by MinitrueSecretary — DISMISSED** [review]
backend LGTM what could possibly go wrong

**Review by JinDamanee2544 — APPROVED** [review]
So far, everything looks good 👍🏻

**Review by Polapob — APPROVED** [review]

**Review by Pacharaponarp — DISMISSED** [review]
LGTM!!!!!!!!

**Review by Polapob — APPROVED** [review]
Look very good to me. I approve

**Review by Polapob — DISMISSED** [review]
LGTM

**Review by Polapob — APPROVED** [review]
Now 01:46 Everything is fine.

**Review by JinDamanee2544 — APPROVED** [review]
So far, everything looks good 👍🏻

**Review by jKulrativid — COMMENTED** [review]
APPROVED

**Review by jKulrativid — APPROVED** [review]

**Pull request merged**